jQuery/styles.css

```diff
@@ -1,27 +1,19 @@
-body {
-  margin: 0;
-  padding: 0;
-}
-
 h1 {
-  margin: 20px 5%;
+  font-size: 2em;
+  margin-bottom: 20px;
 }
 
 #hh-menu {
   border-right: solid 1px black;
   position: fixed;
   width: 240px;
   height: 100%;
-}
 
-#hh-menu > select {
-  width: 80%;
-  margin-left: 15%;
-  margin-right: 5%;
+  padding: 20px;
 }
 
-#hh-menu > label {
-  margin-left: 5%;
+.menu-select {
+  width: 90%;
 }
 
 #hh-content {
```

Comment:
Ah. I'm not exactly running this locally just yet, so it's hard for me to know what it's supposed to look like. my b. Still something to think about though